datasets/il/mod_crypto/il_mod_crypto.yml

```diff
@@ -0,0 +1,28 @@
+title: Israel Sanctioned Crypto Wallets List
+entry_point: crawler.py
+prefix: il-nbctf
+coverage:
+  frequency: daily
```

Comment:
these google sheets ones we set frequency to `never` and then add a note about it in the description as in https://www.opensanctions.org/datasets/lt_illegal_websites/

Reply:
Thanks, done!